contracts/Vault.vy

```diff
@@ -1,4 +1,4 @@
-# @version 0.2.11
+# @version 0.2.12
```

Comment:
Must target an Vault release. Can you provide a summary of codesize and gas cost changes between 0.2.11 and 0.2.12 using our test suite?